docs/contributing/dependencies/index.md

```diff
@@ -0,0 +1,116 @@
+---
+sidebar_custom_props:
+  access: bitwarden
+---
+
+# Dependency Management
+
+Bitwarden uses [Renovate](https://www.mend.io/renovate/) for automating dependency updates. Renovate
+will automatically create pull requests for dependencies on a weekly cadence. Security updates will
+generate pull requests immediately.
+
+## Ownership
+
+Bitwarden's repositories falls under two categories, team-owned and shared repositories.
+
+### Team-Owned repositories
+
+Team owned repositories are "owned" by a single team from a dependency standpoint. The assigned team
+is responsible for reviewing, approving and merging dependency updates. Some reason a repository
+might be team-owned is because it's primarily developed by that team, or to balance out the number
+of dependencies teams have to manage.
+
+Some examples are [`directory-connector`][dc] which is owned by the _Admin Console_ team and
+[`key-connector`][kc] which is owned by the _Auth_ team.
+
+### Shared repositories
+
+Shared repositories don't have any direct owner, instead each dependency is allocated to a team. The
+team assigned to a dependency is responsible for reviewing, approving and merging that dependency.
+For major upgrades the team is responsible for coordinating the upgrade with the other teams.
+
+Examples of shared repositories are [`server`][server] and [`clients`][clients].
+
+## Example PR
+
+<figure>
+
+![Screenshot of a  Renovate PR](./renovate-pr.png)
+
+<figcaption>Example Renovate PR</figcaption>
+
+</figure>
+
+Renovate PRs contains several areas of interest. In the example PR we can see that it contains two
+dependencies, which are grouped together. The PR proposes to upgrade the dependencies from `6.0.21`
+to `7.0.12`. The age of the version is **13 days**, and **13%** of repositories have adopted this
+version. Renovate has seen a **74%** test success rate across Renovate managed repositories and have
+a low confidence in the change. For more details read
+[Renovate documentation about Merge Confidence](https://docs.renovatebot.com/merge-confidence/).
+
+## Workflow
+
+Renovate will automatically create pull requests during the weekend, this naturally aligns with each
```

Comment:
```suggestion
Renovate will automatically create pull requests during the weekend as this naturally aligns with each
```

Reply:
I changed it to `Renovate will automatically create pull requests during the weekend, which naturally aligns with
each team allocating some time during the following monday to work through their respective queue of
pull requests.`